### PR TITLE
Add Skip Logic section to xlsx converter reference

### PR DESCRIPTION
**Pull request by EstherPhebe**
addresses odk-x/tool-suite-X#207

#### What is included in this PR?

1. Add Skip Logic section to XLSX Converter in ODK-X Documentation.

### CODE REVIEW

File: odkx-src/xlsx-converter-using.rst [hunk 1]
```diff
@@ -144,6 +144,44 @@ Now, instead of typing their age, the user simply selects whether they are older
 
   Because you determine whether a question is :tc:`select_one` or :tc:`select_multiple` from the **survey** worksheet, the same choice set on the **choices** worksheet can be used for both :tc:`select_one` and :tc:`select_multiple` questions.
 
+.. _xlsx-using-skip-logic:
+
+Using Skip Logic
+-------------------------------------
+
+Skip Logic (*Conditional Branching*) is an amazing feature in :doc:`survey-using` that changes the next prompt or screen the user sees based on their current response. This enables the form creator give users a unique and personalized experience depending on their answers throughout the entire survey.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-using.rst:152]: "This enables the form creator to" to missing

Comment (elmps2018) [reply odkx-src/xlsx-converter-using.rst:152]: Not sure Logic needs caps, probably just "Skip logic" here and elsewhere.

File: odkx-src/xlsx-converter-using.rst [hunk 2]
```diff
@@ -144,6 +144,44 @@ Now, instead of typing their age, the user simply selects whether they are older
...
   Because you determine whether a question is :tc:`select_one` or :tc:`select_multiple` from the **survey** worksheet, the same choice set on the **choices** worksheet can be used for both :tc:`select_one` and :tc:`select_multiple` questions.
 
+.. _xlsx-using-skip-logic:
+
+Using Skip Logic
+-------------------------------------
+
+Skip Logic (*Conditional Branching*) is an amazing feature in :doc:`survey-using` that changes the next prompt or screen the user sees based on their current response. This enables the form creator give users a unique and personalized experience depending on their answers throughout the entire survey. 
+
+Skip Logic is implemented using if, else and end if tags. It uses the **clause** and **condition** columns in the survey worksheet, the if, else and end if tags are placed in the clause column and the conditional expressions are housed in the condition column.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-using.rst:154]: also "else if"

File: odkx-src/xlsx-converter-using.rst [hunk 3]
```diff
@@ -144,6 +144,44 @@ Now, instead of typing their age, the user simply selects whether they are older
...
+
+Below is the survey worksheet of an example survey that implements skip logic. ODK-X has built-in `JavaScipt Operators <https://docs.odk-x.org/xlsx-converter-reference/#javascript-operators>`_ which are useful for creating complex conditional expressions.
+
+.. csv-table:: Using Skip Logic Example Survey Worksheet
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-using.rst:158]: condition should be lower case.

File: odkx-src/xlsx-converter-using.rst [hunk 4]
```diff
@@ -144,6 +144,44 @@ Now, instead of typing their age, the user simply selects whether they are older
...
+  "end if",
+  "if", "selected(data('menu'), 'cupcake')",
+  "begin screen",
+  ,,"select_one", "box_list", "box", "What would you like?"
+  ,,"select_one", "flavor_list", "flavor", "What cake flavor would you like?"
+  "end screen",
+  "end if",
+  ,,"note", "<h3>Order Summary for <u>{{data.name}}</u></h3>"
+
+In the example worksheet above, the respondent can pick through a number of pastries available and select one option. The next set of questions that follow are fully dependent on the choice the user makes. For example, if the user picked Cakes; they are asked for a flavor and size they would like where as if the user picked Doughnuts; they are asked what size of box they would like.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-using.rst:180]: whereas not where as

File: odkx-src/xlsx-converter-using.rst [hunk 5]
```diff
@@ -144,6 +144,44 @@ Now, instead of typing their age, the user simply selects whether they are older
...
+
+.. note::
+  An important thing to remember when using the clause column is the opening **if** tag and the closing **end if** tags. The general rule is that all **if** tags must have a corresponding closing **end if** tag.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-using.rst:183]: Another important note is that you need to make sure your requireds make sense with your ifs if you are using both.

**Review by elmps2018 — COMMENTED** [review]
Awesome to get this documented! Some minor comments.

File: odkx-src/xlsx-converter-using.rst [hunk 6]
```diff
@@ -177,11 +177,16 @@ Below is the survey worksheet of an example survey that implements skip logic. O
...
 
-In the example worksheet above, the respondent can pick through a number of pastries available and select one option. The next set of questions that follow are fully dependent on the choice the user makes. For example, if the user picked Cakes; they are asked for a flavor and size they would like where as if the user picked Doughnuts; they are asked what size of box they would like.
+In the example worksheet above, the respondent can pick through a number of pastries available and select one option. The next set of questions that follow are fully dependent on the choice the user makes. For example, if the user picked Cakes; they are asked for a flavor and size they would like whereas if the user picked Doughnuts; they are asked what size of box they would like.
 
 .. note::
   An important thing to remember when using the clause column is the opening **if** tag and the closing **end if** tags. The general rule is that all **if** tags must have a corresponding closing **end if** tag.
 
+.. note::
+  Do know that if you are using required along with Skip logic, it is important that both the required and if conditionals align.
```
Comment (elmps2018) [inline odkx-src/xlsx-converter-using.rst:186]: Super tiny edit -- skip lower case.

**Review by elmps2018 — COMMENTED** [review]
Very helpful changes! One tiny edit.

**Review by elmps2018 — APPROVED** [review]
Looks great to me!

**Review by wbrunette — APPROVED** [review]

**Pull request merged**